README.md

```diff
@@ -16,6 +16,16 @@ All examples are provided in Kotlin :+1:
 
 ### Basics
 
-[**BasicAnalysis**](https://github.com/BlendVision/Android-Player-SDK/tree/develop/BasicAnalysis): Demonstrates the configuration for sending analytics events for basic analysis.  
-[**BasicCasting**](https://github.com/BlendVision/Android-Player-SDK/tree/develop/BasicCasting): Explains how to stream video and audio directly to a TV.  
-[**BasicPlayback**](https://github.com/BlendVision/Android-Player-SDK/tree/develop/BasicPlayback): Illustrates the setup for basic playback of DASH streams with DRM support using UniPlayer.  
+[**BasicAnalysis**](https://github.com/BlendVision/Android-Player-SDK/tree/develop/BasicAnalysis):
+Demonstrates the configuration for sending analytics events for basic analysis.  
+[**BasicCasting**](https://github.com/BlendVision/Android-Player-SDK/tree/develop/BasicCasting):
+Explains how to stream video and audio directly to a TV.  
+[**BasicPlayback**](https://github.com/BlendVision/Android-Player-SDK/tree/develop/BasicPlayback):
+Illustrates the setup for basic playback of DASH streams using UniPlayer.
+
+### Advance
+
+[**AdvancePlayback**](https://github.com/BlendVision/Android-Player-SDK/tree/develop/AdvancePlayback): Illustrates the
```

Comment:
Did you mean "Advanced"?

- advance: completed before necessary / preceding
- advanced: involving greater complexity; more difficult, elaborate or specialized

Reply:
@dk00 sorry....
I made a mistake. I've already corrected it.